src/wizard/javascript/sveltekit/with-error-monitoring-and-performance.md

```diff
@@ -0,0 +1,66 @@
+---
+name: SvelteKit
+doc_link: https://docs.sentry.io/platforms/javascript/guides/sveltekit/
+support_level: production
+type: framework
+---
+
+## Install
+
+Configure your app automatically with the [Sentry wizard](https://docs.sentry.io/platforms/javascript/guides/sveltekit/#install).
+
+```bash
+npx @sentry/wizard -i sveltekit
+```
+
+## Configure
+
+Sentry wizard will automatically patch your application to configure the Sentry SDK:
+
+- Create or update `src/hooks.client.js` and `src/hooks.server.js` with the default `Sentry.init` call and SvelteKit hooks handlers.
+- Update `vite.config.js` to add source maps upload and auto-instrumentation via Vite plugins.
+- create `.sentryclirc` and `sentry.properties` files with configuration for sentry-cli (which is used when automatically uploading source maps).
+
+Alternatively, you can also [set up the SDK manually](https://docs.sentry.io/platforms/javascript/guides/sveltekit/manual-setup/).
+
+**Configure the Sentry SDK**:
+
+To configure the Sentry SDK on the client-side, edit the `Sentry.init` options in `hooks.client.(js|ts)`:
+
+```javascript
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  // Performance Monitoring
+  integrations: [new Sentry.BrowserTracing()],
+  tracesSampleRate: 1.0, // Capture 100% of the transactions. Adjust this value in production as necessary.
+});
+```
+
+For the server-side, edit the `Sentry.init` options in `hooks.server.(js|ts)`:
+
+```javascript
+import * as Sentry from '@sentry/sveltekit';
+
+Sentry.init({
+  dsn: '___PUBLIC_DSN___',
+  tracesSampleRate: 1.0, // Capture 100% of the transactions. Adjust this value in production as necessary.
```

Comment:
Oof good catch Priscila, thanks! 😅 The `BrowserTracing` integration doesn't need to be set at all because we include it by default for our full stack SDKs (Sveltekit, NextJs, Remix). Will update the snippets!